### PR TITLE
Security: change Boolean to xsd:boolean and fix justification type

### PR DESCRIPTION
**Pull request by armintaenzertng**
While working with the Security profile, I noticed these small issues.

@tsteenbe: I also noticed that [VexStatusType](https://github.com/spdx/spdx-3-model/blob/main/model/Security/Vocabularies/VexStatusType.md) is unused; is this obsolete or was it forgotten to add to a class?

### CODE REVIEW

**Review by kestewart — APPROVED** [review]
LGTM

**Review by goneall — APPROVED** [review]
LGTM - Thanks @armintaenzertng

File: model/Security/Classes/VexNotAffectedVulnAssessmentRelationship.md [hunk 1]
```diff
@@ -48,13 +48,13 @@ for VEX.
 ## Metadata
 
 - name: VexNotAffectedVulnAssessmentRelationship
-- SubclassOf:  VexVulnAssessmentRelationship
+- SubclassOf: VexVulnAssessmentRelationship
 - Instantiability: Concrete
 
 ## Properties
 
 - justification
```
Comment (jeff-schutt) [inline model/Security/Classes/VexNotAffectedVulnAssessmentRelationship.md:56]: Suggest changing this name for consistency across the security properties that use vocabularies.
```suggestion
- justificationType
```

Comment (jeff-schutt) [reply model/Security/Classes/VexNotAffectedVulnAssessmentRelationship.md:56]: For consistency we should also change `Security/Classes/SsvcVulnAssessmentRelationship` Properties from `decision` to `decisionType`

**Review by jeff-schutt — CHANGES_REQUESTED** [review]
> @tsteenbe: I also noticed that [VexStatusType](https://github.com/spdx/spdx-3-model/blob/main/model/Security/Vocabularies/VexStatusType.md) is unused; is this obsolete or was it forgotten to add to a class?

IIRC this is an obsolete leftover from an outdated model diagram b/c these definitions are now found at /model/Core/Vocabularies/RelationshipType.md. @puerco do you agree?

Great work @armintaenzertng!

File: model/Security/Classes/VexNotAffectedVulnAssessmentRelationship.md (hunk 1 above)
Comment (armintaenzertng) [reply model/Security/Classes/VexNotAffectedVulnAssessmentRelationship.md]: Thanks for the suggestions, @jeff-schutt, I included them in the two new commits.

Comment (jeff-schutt) [reply model/Security/Classes/VexNotAffectedVulnAssessmentRelationship.md]: LGTM

**Review by jeff-schutt — APPROVED** [review]

**Review by puerco — APPROVED** [review]
Thanks @armintaenzertng `decisionType` and `justificationType` feel much more consistent with the overall model.

**Comment by puerco** [comment]
> I also noticed that `VexStatusType` is unused; is this obsolete or was it forgotten to add to a class?

Yep, the status vocabulary is no longer used as now the status is a relationship, thanks for catching that

**Pull request merged**